docs/source/using/basic-usage.adoc

```diff
@@ -177,9 +177,12 @@ This environment variable is currently experimental and semantics might change i
 === Persistent Configuration
 
 Using persistent configuration allows you to control {project} behavior without specifying actual command line flags, similar to the way you use environment variables.
+You can specify persistent configuration as global (using `global` flag) to make each profile consumes it without setting it for individual profile.
```

Comment:
suggestion: `You can also define global configuration using `--global` flag. Global configuration is applicable to all profiles.`